src/components/About.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+class About extends Component {
+  render() {
+   return (
+    <div>
+      <h2>About</h2>
```

Comment:
including other places, could you fix indent? 
I hope you fix along eslint rules as far as possible. :pray: